lib/codegen/meta-python/base/instructions.py

```diff
@@ -144,6 +145,34 @@
         """,
         ins=(x, JT), is_branch=True)
 
+jump_table_entry = Instruction(
+    'jump_table_entry', r"""
+    Get an entry from a jump table.
+
+    Load an address from a specific entry in a jump table.
+    """,
+    ins=(x, JT), outs=addr)
```

Comment:
Wording nit: the value being loaded isn't an "address"; in the current patch it's a relative offset from the start of the table (in the future it could be other things too).